nixos/modules/services/hardware/udisks2.nix

```
@@ -19,7 +18,18 @@ in
 
     services.udisks2 = {
 
-      enable = mkEnableOption (lib.mdDoc "udisks2, a DBus service that allows applications to query and manipulate storage devices");
+      enable = mkEnableOption (mdDoc "udisks2, a DBus service that allows applications to query and manipulate storage devices");
+
+      mountRoot = mkOption {
+        type = types.bool;
+        default = false;
+        description = mdDoc ''
+          By default, udisks2 mounts removable drives under the ACL controlled directory `/run/media/$USER/`.
+          If you wish to mount under `/media` instead, enable this option. This feature will also clean `/media`
+          directory at every boot. Since `/media`, unlike `/run`, is not mounted by default as a tmpfs, it
+          needs to be cleaned to get rid of stale mountpoints.
```

```suggestion
          When enabled, instructs udisks2 to mount removable drives under `/media/` directory, instead of the default, ACL-controlled `/run/media/$USER/`.

          Since `/media/` is not mounted as tmpfs by default, it requires cleanup to get rid of stale mountpoints; enabling this option will take care of this at boot.
```

(It happens at boot, or at _re_ boot?)

---

It happens at every boot, hard boot or reboot.